resources/views/auth/login.blade.php

```diff
@@ -42,6 +42,6 @@
                 </a>
             @endif
         </div>
-       
+
```

Comment:
Before opening a PR, try to review your code yourself, see if there are any unnecessary commits.

In this file you are versioning a space